### PR TITLE
Issue #9880: update ArrayTypeStyleCheckTest to use unique Input file …

### LINKED ISSUE #9880: update ArrayTypeStyleCheckTest to use unique Input file in each test method
ArrayTypeStyleCheckTest.java file reuse some inputs in few different test method.
Expected: each test method in Test class use it own Input file and do not share it with any other test method.

more details - https://github.com/checkstyle/checkstyle/wiki/Good-practice-on-how-to-create-Input-files-for-tests

@ mentors: Can this issue be approved as a gsoc second issue? Please review.

### PR DESCRIPTION
**Pull request by haanhvu**
closes #9880

### CODE REVIEW

**Review by pbludov — APPROVED** [review]

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/arraytypestyle/InputArrayTypeStyle.java [hunk 1]
```diff
@@ -4,21 +4,20 @@
 ////////////////////////////////////////////////////////////////////////////////
 package com.puppycrawl.tools.checkstyle.checks.arraytypestyle;
 
-/**
- * Test case for ArrayTypeStyle (Java vs C)
- * @author lkuehne
- **/
+/* Config:
+ * javaStyle = true
```
Comment (timurt) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/arraytypestyle/InputArrayTypeStyle.java:8]: You can just write `Config: default`, if no properties were specified

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/ArrayTypeStyleCheckTest.java [hunk 2]
```diff
@@ -68,15 +68,15 @@ public void testJavaStyleOff()
...
         checkConfig.addAttribute("javaStyle", "false");
         final String[] expected = {
-            "13:16: " + getCheckMessage(MSG_KEY),
-            "17:39: " + getCheckMessage(MSG_KEY),
-            "23:18: " + getCheckMessage(MSG_KEY),
-            "31:20: " + getCheckMessage(MSG_KEY),
-            "45:33: " + getCheckMessage(MSG_KEY),
-            "50:36: " + getCheckMessage(MSG_KEY),
-            "56:29: " + getCheckMessage(MSG_KEY),
+            "12:16: " + getCheckMessage(MSG_KEY),
```
Comment (timurt) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/ArrayTypeStyleCheckTest.java:71]: input has 4 `// violation` comments but 7 violations are expected, please check and fix

**Review by timurt — CHANGES_REQUESTED** [review]

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/arraytypestyle/InputArrayTypeStyle.java (hunk 1 above)
Comment (haanhvu) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/arraytypestyle/InputArrayTypeStyle.java]: @timurt done

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/ArrayTypeStyleCheckTest.java (hunk 2 above)
Comment (haanhvu) [reply src/test/java/com/puppycrawl/tools/checkstyle/checks/ArrayTypeStyleCheckTest.java:71]: @timurt the first 4 lines are `// violation`, but the last 3 lines are `// ok`

Please check again. I think the last 3 lines are actually in C style.

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/arraytypestyle/InputArrayTypeStyleOff.java [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+        {
+            return null;
+        }
+
+        public Test[] getNewTest()
+        {
+            return null;
+        }
+
+        public Test getOldTest()[] // ok
```
Comment (timurt) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/arraytypestyle/InputArrayTypeStyleOff.java:44]: https://github.com/checkstyle/checkstyle/pull/9986/files#r637582781

here violation is expected

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/arraytypestyle/InputArrayTypeStyleOff.java [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+        {
+            return null;
+        }
+
+        public Test getOldTests()[][] // ok
```
Comment (timurt) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/arraytypestyle/InputArrayTypeStyleOff.java:49]: https://github.com/checkstyle/checkstyle/pull/9986/files#r637582781

here violation is expected

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/arraytypestyle/InputArrayTypeStyleOff.java [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+        {
+            return null;
+        }
+
+        public Test[]
+            getMoreTests()[][] // ok
```
Comment (timurt) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/arraytypestyle/InputArrayTypeStyleOff.java:55]: https://github.com/checkstyle/checkstyle/pull/9986/files#r637582781

here violation is expected

**Review by timurt — CHANGES_REQUESTED** [review]

**Review by timurt — APPROVED** [review]

**Review by romani — APPROVED** [review]

**Pull request merged**